NineChroniclesUtilBackend/Models/Arena/ArenaRanking.cs

```diff
@@ -0,0 +1,30 @@
+using NineChroniclesUtilBackend.Models.Agent;
+
+namespace NineChroniclesUtilBackend.Models.Arena;
+
+public class ArenaRanking(
```

Comment:
It seems like a case to use `record` type. (maybe)

Reply:
At first, I thought adding cp calculation part to getter (or static method) of ArenaRanking or Avatar. Because it's kind of (Domain) Entity and cp calculation code is domain logic. I will apply these refactor on following Pull Request and ticket.